main.go

```diff
@@ -472,10 +472,15 @@ func aggregateTimemap(urir string, dttmp *time.Time, sess *Session) (basetm *lis
 }
 
 func parseURI(uri string) (urir string, err error) {
-	if !regs["isprtcl"].MatchString(uri) {
-		uri = "http://" + uri
+	uescd, err := url.PathUnescape(uri)
+	if err != nil {
+		logError.Printf("Error Unescaping path (%s): %v", uri, err)
+		return
+	}
```

Comment:
```suggestion
	}
	uescd = strings.ReplaceAll(uescd, " ", "%20")
```

Reply:
I did this and the memento from 2013 appeared. The changes are reflected in the latest commit of this branch/PR e6e7820

```
curl -v "http://localhost:1208/timemap/link/https://google.com/?q=united%20states"
*   Trying 127.0.0.1:1208...
* Connected to localhost (127.0.0.1) port 1208 (#0)
> GET /timemap/link/https://google.com/?q=united%20states HTTP/1.1
> Host: localhost:1208
> User-Agent: curl/8.1.2
> Accept: */*
> 
< HTTP/1.1 200 OK
< Access-Control-Allow-Origin: *
< Access-Control-Expose-Headers: Link, Location, X-Memento-Count, Server
< Content-Type: application/link-format
< Server: MemGator/1.0-rc8
< X-Memento-Count: 1
< Date: Wed, 17 Apr 2024 18:55:11 GMT
< Content-Length: 780
< 
<https://google.com/?q=united%20states>; rel="original",
<http://localhost:1208/timemap/link/https://google.com/?q=united%20states>; rel="self"; type="application/link-format",
<https://web.archive.org/web/20131212220548/http://www.google.com/?q=United%2520States>; rel="first last memento"; datetime="Thu, 12 Dec 2013 22:05:48 GMT",
<http://localhost:1208/timemap/link/https://google.com/?q=united%20states>; rel="timemap"; type="application/link-format",
<http://localhost:1208/timemap/json/https://google.com/?q=united%20states>; rel="timemap"; type="application/json",
<http://localhost:1208/timemap/cdxj/https://google.com/?q=united%20states>; rel="timemap"; type="application/cdxj+ors",
<http://localhost:1208/timegate/https://google.com/?q=united%20states>; rel="timegate"
```